Update hcmarcus.msg

fedprod opened the pull request:
fix for entry {1100} that has a french speech sample in the german version

burner1024 commented:
Sorry, but translation updates are to be submitted only through [translation system](https://tra.bgforge.net/projects/fallout/up/).

As for speech sample, there's no speech in this repo, so it fixes nothing. If you wanted to do a fix, it would require adding the sound files first, and then renaming the corresponding file.

fedprod commented:
Are you seriously kidding me? I changed it to a complete other speech sample that is already in the game!
The second parenthesis are used to reference an ACM file! So I did not change a soundfile because I rewrote the line!
As fore using weblate, as long as you are not going to make changes to it that one can **_edit a specific file AND string_** and also can edit the entry for the second parenthis people are not going to use it!

burner1024 commented:
I don't know what's in your game. In this repo, there no german, nor french sound.
If a sound sample is wrong, the proper solution is to find the right one and submit it. I suggest to start with that.
If it doesn't exist at all, then options are either leaving it silent or replacing with another, like you did. However, like I said, any changes to translations go through the system.

Lexx2k commented:
The german sound file is in the german master.dat file. No point in uploading it somewhere.

burner1024 commented:
Then the best thing to do is to change the translation and copy the other acm under a new name.

Lexx2k commented:
Ehhh why?

burner1024 commented:
That really depends on the point of view.
Anyway, flexibility is good when it serves some purpose other than itself.
